packages/tldraw/src/lib/tools/SelectTool/childStates/Translating.ts

```diff
@@ -338,7 +340,7 @@ function getTranslatingSnapshot(editor: Editor) {
 	}
 
 	return {
-		averagePagePoint: Vec.Average(pagePoints),
+		averagePagePoint: pagePoints.length ? Vec.Average(pagePoints) : new Vec(0, 0),
```

Comment:
The Vec.Average should handle this case

Reply:
[Now it does](https://github.com/tldraw/tldraw/pull/3065/files#diff-e8cd5987588f3ef008d49f455970555cd1a231028a771f1be7251deaf3638cdcR534-R536), before it called `div` with 0.